BREAKING.md

```diff
@@ -29,11 +29,11 @@ The following public API functions on `Loader`, from `"@fluidframework/container
 
 All of the required functionality from a `Container` instance should be available on `IContainer`. If the function or property you require is not available, please file an issue on GitHub describing which function and what you are planning on using it for. They can still be used by casting the returned object to `Container`, i.e. `const container = await loader.resolve(request) as Container;`, however, this should be avoided whenever possible and the `IContainer` API should be used instead.
 
-### `getQuorum()` returns `IQuorumClients` from within the container
+### `getQuorum()` and `quorum` members return `IQuorumClients`
```

Comment:
0.55 bump happened last week so it needs a new section

Reply:
Ahh, thanks, I'm so last year